client/checkout/classic/test/payment-processing.test.js

```diff
@@ -347,7 +347,25 @@ describe( 'Payment processing', () => {
 			}
 		} );
 	} );
+
 	afterEach( () => {
+		[
```

Comment:
The form elements kept getting added at each test run, but never getting removed from the DOM.
Fixing.